src/renderer/pages/project/script_editor/beat_selector.vue

```
@@ -15,7 +15,8 @@
   </div>
 </template>
 <script setup lang="ts">
-import { computed } from "vue";
+import { computed, onMounted } from "vue";
+import { isNull } from "graphai";
```

ESLint

[error] Remove this unused import of 'isNull'.

(sonarjs/unused-import)